output_template/python/run.py

```diff
@@ -133,19 +155,19 @@ def main_test(input_image, library, config_file, max_percent_incorrect_values=0.
     help="Input image filename",
 )
 @click.option(
-    "-l",
-    "--library",
+    "-m",
+    "--model",
     type=click.Path(exists=True),
-    help="Shared library filename",
+    help="Inference Model filename",
 )
 @click.option(
     "-c",
     "--config_file",
     type=click.Path(exists=True),
     help="Config file Path",
 )
-def run_test(input_image, library, config_file):
-    main_test(input_image, library, config_file)
+def run_test(input_image, model, config_file):
```

Comment:
Also `run_test()` is so bad method name ... 😭

Reply:
🆗 I'll change it to `initialize()` is it okay?

Reply:
OK. 👍 I prefer `main()`.